tui/processtree/processtree.go

```diff
@@ -110,6 +111,44 @@ func NewProcessTree(ctx context.Context, opts []ProcessTreeOption, tree ...*Proc
 		total++
 		item.norender = pt.norender
 		item.timeout = pt.timeout
+
+		if pt.norender {
+			item.ctx = pt.ctx
+			return nil
+		}
+
+		ictx := pt.ctx
+
+		logger, err := kamino.Clone(log.G(ictx),
+			kamino.WithZeroUnexported(),
+		)
+		if err != nil {
+			return err
+		}
+
+		logger.Out = item
+
+		if formatter, ok := logger.Formatter.(*log.TextFormatter); ok {
+			formatter.ForceColors = termenv.DefaultOutput().ColorProfile() != termenv.Ascii
+			formatter.ForceFormatting = true
+			logger.Formatter = formatter
+		}
+
+		ictx = log.WithLogger(ictx, logger)
+
+		ios, err := kamino.Clone(iostreams.G(ictx),
+			kamino.WithZeroUnexported(),
+		)
+		if err != nil {
+			return err
+		}
+
+		ios.Out = iostreams.NewNoTTYWriter(item, iostreams.G(ctx).Out.Fd())
+		ios.In = iostreams.G(ctx).In
```

Comment:
What abous `ios.Err`?

Reply:
Makes sense. This hasn't affected my testing so far and adding it hasn't caused any issues, so I've now forced pushed with an updated.